apache/trafficcontrol
# Add comment field to parameters

Closes: #7456 

Add field "comment" to parameter

## Which Traffic Control components are affected by this PR?
- Traffic Ops
- Traffic Portal

## What is the best way to verify this PR?
- Enable show parameters "comment" field in traffic_portal_properties.json: properties.parameters.comment.show = true
- Provisioning CDN in a Box, select Configure/Parameters, 1 more columns "comment" should show
- Create/Update Parameters with "comment"

## If this is a bugfix, which Traffic Control versions contained the bug?


## PR submission checklist
